File: examples/hello-python/hello.py
#!/usr/bin/env python3
import sys
from datetime import datetime

name = sys.argv[1] if len(sys.argv) > 1 else "World"
print(f"Hello, {name}! 🐍")
print(f"Generated at: {datetime.now().isoformat()}")
print(f"Python version: {sys.version.split()[0]}")

File: examples/hello-go/hello.go
package main

import (
	"fmt"
	"os"
	"runtime"
)

func main() {
	name := "World"
	if len(os.Args) > 1 {
		name = os.Args[1]
	}
	fmt.Printf("Hello, %s! 🐹\n", name)
	fmt.Printf("Go version: %s\n", runtime.Version())
}
